### PR TITLE
PR

### PR DESCRIPTION
**Pull request by oscarmild**

### CODE REVIEW

File: frontend/deploy/build.sh [hunk 1]
```diff
@@ -1 +1,6 @@
 
+
```
Comment (fromgopi) [inline frontend/deploy/build.sh:2]: Adding more empty space will not be good

Comment (oscarmild) [reply frontend/deploy/build.sh:2]: Ok Will remove all the empty spaces in repo

File: frontend/webpack.config.js [hunk 2]
```diff
@@ -16,4 +16,6 @@ module.exports = {
 			
 		}
 	}
-};
\ No newline at end of file
+};
+
```
Comment (fromgopi) [inline frontend/webpack.config.js:20]: @oscarmild We have to add more items for solving css and images

**Review by fromgopi — APPROVED** [review]
@oscarmild Approved. Please close the comment. I will merge the pr

File: frontend/webpack.config.js (hunk 2 above)
Comment (fromgopi) [reply frontend/webpack.config.js:20]: Ok

**Pull request merged**